libsolidity/interface/StandardCompiler.cpp

```diff
@@ -740,67 +736,69 @@ std::variant<StandardCompiler::InputsAndSettings, Json::Value> StandardCompiler:
 					return formatFatalError("JSONError", "Invalid hex encoding of SMTLib2 auxiliary input.");
 				}
 
-				if (!smtlib2Responses[hashString].isString())
+				if (!response.is_string())
 					return formatFatalError(
 						"JSONError",
 						"\"smtlib2Responses." + hashString + "\" must be a string."
 					);
 
-				ret.smtLib2Responses[hash] = smtlib2Responses[hashString].asString();
+				ret.smtLib2Responses[hash] = response.get<string>();
 			}
 		}
 	}
 
-	Json::Value const& settings = _input.get("settings", Json::Value());
+//	if (!_input.contains("settings"))
+//		_input["settings"] = Json::object();
+	Json const& settings = _input["settings"];
```

Comment:
So there's no `get()` with a default?

Reply:
Yes, that's one of the questions how to do that nicely.

Reply:
We could just create a helper function for that.

Reply:
There's actually `.value(key, default)` instead. Used that now in most places.